### PR TITLE
Add Geekbench ML 0.6.0

### PR DESCRIPTION
**Pull request by owine**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/g/geekbench-ml.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "geekbench-ml" do
+  version "0.6.0"
+  sha256 "ae577f44e397cb1fea8873c72bb317f9145cca9ec9d6cfa79bad866f287500a3"
+
+  url "https://cdn.geekbench.com/GeekbenchML-#{version}-Mac.zip"
+  name "geekbench-ml"
```
Comment (bevanjkay) [inline Casks/g/geekbench-ml.rb:6]:
```suggestion
  name "Geekbench ML"
```

File: Casks/g/geekbench-ml.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    url "https://www.primatelabs.com/appcast/geekbenchml.xml"
+    strategy :sparkle, &:short_version
+  end
+
+  auto_updates true
+  depends_on macos: ">= :ventura"
+
+  app "Geekbench ML.app"
+
+  # No zap stanza required
```
Comment (bevanjkay) [inline Casks/g/geekbench-ml.rb:20]: There is likely to be some kind of preference files. What process did you use to try to find files?

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/g/geekbench-ml.rb (hunk 2 above)
Comment (owine) [reply Casks/g/geekbench-ml.rb:20]: `brew createzap`

Comment (bevanjkay) [reply Casks/g/geekbench-ml.rb:20]: What did you pass as the argument to `brew createzap`?

Comment (owine) [reply Casks/g/geekbench-ml.rb:20]: `brew createzap geekbench-ml`

Comment (bevanjkay) [reply Casks/g/geekbench-ml.rb:20]: The developer may have used a different bundle id for the files. Can you try `brew createzap geekbench` and see if anything is returned?

Comment (owine) [reply Casks/g/geekbench-ml.rb:20]:
```
➜  homebrew-cask git:(geekbench-ml) brew createzap geekbench-ml
No geekbench-ml settings found.
```

Comment (owine) [reply Casks/g/geekbench-ml.rb:20]: > The developer may have used a different bundle id for the files. Can you try `brew createzap geekbench` and see if anything is returned?

Our posts crosed, for just `geekbench` a zap block is returned.

**Comment by bevanjkay** [comment]
Thanks @owine 🚀

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**